packages/ui/lib/RadioGroup/context.ts

```diff
@@ -0,0 +1,8 @@
+import { createContext } from "react";
+
+export type RadioGroupContext = {
+  value: string;
+  onChange: (value: string) => void;
```

Comment:
I'm wondering if we should expose the original event here.

```tsx
onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

and maybe an additional callback to get the value directly

```tsx
onChangeValue: (value: string) => void;
```

Maybe the same would be handy for all inputs, without having to query `event.targe.value` it is possible to:

```tsx
<InputText onChangeValue={setValue} />
```

But sometimes the original event might be needed.

Reply:
What about as a second param? (value: string, e: ChangeEvent<HTMLInputElement>)=>void